src/ui/pages/zcl_abapgit_gui_page_data.clas.abap

```diff
@@ -94,6 +102,30 @@ ENDCLASS.
 CLASS zcl_abapgit_gui_page_data IMPLEMENTATION.
 
 
+  METHOD get_form_schema.
```

Comment:
this duplicates the code in `render_add()` ?

Reply:
Thank you for pointing that out. I plan to refactor the code soon. I'll keep you updated once the refactoring is completed.

Reply:
Hi Lars,

The method render_add was removed since it is not needed anymore. Now there is no duplicate code.

Thanks,

Francisco.